client/my-sites/marketing/connections/service-examples.jsx

```diff
@@ -4,6 +4,7 @@ import { includes } from 'lodash';
 import PropTypes from 'prop-types';
 import { Component } from 'react';
 import { connect } from 'react-redux';
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
```

Comment:
The thing is that even with Jetpack sites, the links won't work because it wants to go to the marketing section, which is only available in Calypso, not in Cloud. So what we could do is to check if the site is Jetpack site, and if it is not, hide the section as now. 

If it is however there are two scenarios:
* we are in JP Cloud: We should point to JP support, or calculate the Wordpress.com links somehow as you said above.
* we are in Calypso: Show the calculated links as before

What do you think? @spsiddarthan @pablinos